js-createElement/js-createelement_social-media-post/js/index.js

```diff
@@ -10,3 +10,34 @@ likeButton.addEventListener("click", handleLikeButtonClick);
 
 // Exercise:
 // Use document.createElement() and append another social media post to the body.
+
+const newPostSection = document.createElement("section");
+newPostSection.className = "post";
+
+const newPostContent = document.createElement("p");
+newPostContent.className = "post__content";
+newPostContent.textContent =
+  "This is another social media post created with JavaScript.";
+
+const newPostFooter = document.createElement("footer");
+newPostFooter.className = "post__footer";
+
+const newPostUsername = document.createElement("span");
+newPostUsername.className = "post__username";
+newPostUsername.textContent = "@newuser";
+
+const newLikeButton = document.createElement("button");
+newLikeButton.className = "post__button";
+newLikeButton.setAttribute("type", "button");
+newLikeButton.setAttribute("data-js", "like-button");
+newLikeButton.textContent = "♥ Like";
+
+newLikeButton.addEventListener("click", handleLikeButtonClick);
+
+newPostFooter.appendChild(newPostUsername);
+newPostFooter.appendChild(newLikeButton);
```

Comment:
It is possible to merge and add several elements separated by commas `newPostFooter.appendChild(newPostUsername, newLikeButton);`